### DISCUSSION TITLE
Can Nextjs on-demand revalidation work with aws or any other provider instead of vercel?

### DISCUSSION THREAD

**Discussion by HarshBoricha404** [vercel/next.js#36340] [Help] [answered]
I was watching [this](https://www.youtube.com/watch?v=mMHssutD9mU&ab_channel=codedamn ) video by codedamn. There he said this feature is only for project deployed in vercel. It does not provide netlify, aws, etc.

Is this true?

**Comment by timneutkens** [comment, marked as answer]
On-demand revalidation is not specific to Vercel. It's supported for `next start` as are all other Next.js features.
